pages/index.vue

```diff
@@ -149,10 +149,10 @@
     "相談の手順を見る": "Steps for receiving help",
     "検査陽性者の状況": "Details of confirmed cases",
     "陽性患者数": "Number of confirmed cases",
-    "陽性患者の属性": "Positive patient attributes",
-    "検査実施数": "Number of tested",
-    "新型コロナコールセンター相談件数": "Number of reports to COVID-19 telephone advisory center",
-    "新型コロナ受診相談窓口相談件数": "Number of reports to combined telephone advisory center",
+    "陽性患者の属性": "Confirmed patient attributes",
+    "検査実施数": "Number of tests conducted",
+    "新型コロナコールセンター相談件数": "Number of inquiries to COVID-19 telephone advisory center",
+    "新型コロナ受診相談窓口相談件数": "Number of inquiries to combined telephone advisory center",
```

Comment:
Please see https://docs.google.com/spreadsheets/d/1aksSfWIXE_L3rFYFjHz9xtVhh0n7ealMQ0eEz8GGPlw/edit#gid=0 Row 27 and 152, there are new wording.

Reply:
thank you. I added comment there.